pkg/cmd/workspace/create/cmd.go

```diff
@@ -0,0 +1,44 @@
+package create
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	"kusionstack.io/kusion/pkg/cmd/util"
+	"kusionstack.io/kusion/pkg/util/i18n"
+)
+
+func NewCmd() *cobra.Command {
+	var (
+		short = i18n.T(`Create a new workspace`)
+
+		long = i18n.T(`
+		This command creates a workspace with specified name and configuration file, where the file must be in the YAML format.`)
+
+		example = i18n.T(`
+		# Create a new workspace
+		kusion workspace create dev -f dev.yaml`)
```

Comment:
It seems that the `kusion workspace create` method does not provide the functionality of workspace scaffolding templates? Is there any plan to support it in the future?